benches/search_comparison.rs

```diff
@@ -16,6 +17,12 @@ use std::{
     time::Duration,
 };
 
+/// Because benchmarks are builded with linker flag -rdynamic there should be library entry point defined
+/// in all benchmarks. This is only needed when two harnesses are used.
```

Comment:
This feels like a short-coming in tango — shouldn't it be able to recognize if a particular binary has no entry-point for it?

Reply:
Yes, tango executable is the dynamic library (benchmarks) and the executable (the runner) at the same time. To achieve this some linker shenanigans are required. But I was able to pull it of without requiring dummy entry point. This code is already removed. Thank you.